sigs/sig-marketing/social-media-guidelines.md

```diff
@@ -0,0 +1,50 @@
+**<span style="text-decoration:underline;">Assets</span>**
+
+[Argo Proj Social Content Calendar](https://docs.google.com/spreadsheets/d/1ysidts6KNAr2OKS1L4bgMToUFuAmO1xQ__tTQf4kPWk/edit?usp=sharing) (to prevent duplicate posting)
+
+**<span style="text-decoration:underline;">Guidelines*</span>**
+
+_Access_
+
+
+
+* Members of sig-marketing will have access to Argo Proj Social Accounts (LinkedIn, Twitter, and Youtube)
+* All social media posts by members related to the Argo Project whether from official, company or personal accounts must follow the below guidelines.
+* For posts that do not follow guidelines: 2 members from **_different_** organizations must agree that a post should be taken down and then it will be deleted.
```

Comment:
I think it would be good to have some language about outside input before posting, e.g. 

"Posts and content should be approved by a member from a **_different_** organization before being published. "